### PR TITLE
Fix Pipelines E2E testing

### PR DESCRIPTION
**Pull request by diegolovison**
Fix Pipelines E2E testing

```
sh run_robot_test.sh --set-urls-variables true --extra-robot-args '-L DEBUG -i ODS-2632' --skip-oclogin
```

```
Tests.Data Science Pipelines.Test-Run-Data-Science-Pipelines-Operator-E2E-T...
==============================================================================
[ WARN ] No Prometheus namespace found
Run Data Science Pipelines Operator Integration Tests :: Run Data ... | PASS |
------------------------------------------------------------------------------
Tests.Data Science Pipelines.Test-Run-Data-Science-Pipelines-Opera... | PASS |
1 test, 1 passed, 0 failed
==============================================================================
Tests.Data Science Pipelines                                          | PASS |
1 test, 1 passed, 0 failed
==============================================================================
Tests                                                                 | PASS |
1 test, 1 passed, 0 failed
==============================================================================

```

### CODE REVIEW

File: ods_ci/tests/Tests/1100__data_science_pipelines/1110__test-run-data-science-pipelines-operator-e2e-tests.robot [hunk 1]
```diff
@@ -27,26 +26,21 @@ Run Data Science Pipelines Operator Integration Tests
...
-    ${return_code}    ${output}     Run And Return Rc And Output    cd ${DATA-SCIENCE-PIPELINES-OPERATOR-SDK_DIR} && git checkout -b it_test origin/${DATA-SCIENCE-PIPELINES-OPERATOR-SDK_REPO_BRANCH}
+    Should Be Equal As Integers	   ${return_code}	 0  msg=Unable to clone data-science-pipelines-operator repo ${DATA-SCIENCE-PIPELINES-OPERATOR-SDK_REPO_URL}:${DATA-SCIENCE-PIPELINES-OPERATOR-SDK_REPO_BRANCH}:${DSPO_SDK_DIR}
+    ${return_code}    ${output}     Run And Return Rc And Output    cd ${DSPO_SDK_DIR} && git checkout -b it_test origin/${DATA-SCIENCE-PIPELINES-OPERATOR-SDK_REPO_BRANCH}
     Should Be Equal As Integers	   ${return_code}	 0  msg=Unable to checkout data-science-pipelines-operator
     RHOSi Setup
-    ${rc}    ${out}=    Run And Return Rc And Output    oc new-project ${DSPANAMESPACE}
-    Should Be Equal As Integers	   ${rc}	 0  msg=Cannot create a new project ${DSPANAMESPACE}
-
-Teardown Data Science Pipelines Operator Integration Tests Suite
```
Comment (bdattoma) [inline ods_ci/tests/Tests/1100__data_science_pipelines/1110__test-run-data-science-pipelines-operator-e2e-tests.robot:49]: is the teardown implemented in the https://github.com/red-hat-data-services/ods-ci/pull/1833/files#diff-0639f9a87a08627ad0f0742ca7d12ea4ac9ad2a21072e8d216a04d60f43994b9R29 itself?

File: ods_ci/tests/Tests/1100__data_science_pipelines/1110__test-run-data-science-pipelines-operator-e2e-tests.robot [hunk 2]
```diff
@@ -27,26 +26,21 @@ Run Data Science Pipelines Operator Integration Tests
     ...     ODS-2632    AutomationBug
     ${openshift_api}    Get Openshift Server
     Log    ${openshift_api}
-    ${return_code}    ${output}    Run And Return Rc And Output    cd ${DATA-SCIENCE-PIPELINES-OPERATOR-SDK_DIR} && make integrationtest K8SAPISERVERHOST=${openshift_api} DSPANAMESPACE=${DSPANAMESPACE} KUBECONFIGPATH=${KUBECONFIGPATH}
+    ${return_code}    ${output}    Run And Return Rc And Output    cd ${DSPO_SDK_DIR} && GIT_WORKSPACE=${DSPO_SDK_DIR} sh .github/scripts/tests/tests.sh --rhoai --k8s-api-server-host ${openshift_api} --kube-config ${KUBECONFIGPATH} --dspa-path ${DSPO_SDK_DIR}/tests/resources/dspa.yaml --external-dspa-path ${DSPO_SDK_DIR}/tests/resources/dspa-external.yaml --clean-infra --endpoint-type route
```
Comment (bdattoma) [inline ods_ci/tests/Tests/1100__data_science_pipelines/1110__test-run-data-science-pipelines-operator-e2e-tests.robot:29]: you sure about `.github/scripts/tests/tests.sh`? is that folder an hidden one given that it starts with "."?

Comment (diegolovison) [reply ods_ci/tests/Tests/1100__data_science_pipelines/1110__test-run-data-science-pipelines-operator-e2e-tests.robot:29]: Yes. It is accurate. Initially, all scripts for DSPO were developed for GHA. We are making progress to improve. This script will probably end up in another directory in the near future.

File: ods_ci/tests/Tests/1100__data_science_pipelines/1110__test-run-data-science-pipelines-operator-e2e-tests.robot [hunk 3]
```diff
@@ -27,26 +26,21 @@ Run Data Science Pipelines Operator Integration Tests
     ...     ODS-2632    AutomationBug
```
[the remark this answers is not in the document]
Comment (diegolovison) [reply ods_ci/tests/Tests/1100__data_science_pipelines/1110__test-run-data-science-pipelines-operator-e2e-tests.robot]: Fixed. Thanks

File: ods_ci/tests/Tests/1100__data_science_pipelines/1110__test-run-data-science-pipelines-operator-e2e-tests.robot (hunk 1 above)
Comment (diegolovison) [reply ods_ci/tests/Tests/1100__data_science_pipelines/1110__test-run-data-science-pipelines-operator-e2e-tests.robot:49]: Yes. We delegate to the DSPO test suite.

**Review by bdattoma — APPROVED** [review]

File: ods_ci/tests/Tests/1100__data_science_pipelines/1110__test-run-data-science-pipelines-operator-e2e-tests.robot [hunk 4]
```diff
@@ -24,29 +23,24 @@ Run Data Science Pipelines Operator Integration Tests
     [Tags]
     ...     DataSciencePipelines-Backend
     ...     Tier1
-    ...     ODS-2632    AutomationBug
+    ...     ODS-2632
     ${openshift_api}    Get Openshift Server
     Log    ${openshift_api}
-    ${return_code}    ${output}    Run And Return Rc And Output    cd ${DATA-SCIENCE-PIPELINES-OPERATOR-SDK_DIR} && make integrationtest K8SAPISERVERHOST=${openshift_api} DSPANAMESPACE=${DSPANAMESPACE} KUBECONFIGPATH=${KUBECONFIGPATH}
+    ${return_code}    ${output}    Run And Return Rc And Output    cd ${DSPO_SDK_DIR} && GIT_WORKSPACE=${DSPO_SDK_DIR} sh .github/scripts/tests/tests.sh --rhoai --k8s-api-server-host ${openshift_api} --kube-config ${KUBECONFIGPATH} --dspa-path ${DSPO_SDK_DIR}/tests/resources/dspa.yaml --external-dspa-path ${DSPO_SDK_DIR}/tests/resources/dspa-external.yaml --clean-infra --endpoint-type route
```
Comment (jgarciao) [inline ods_ci/tests/Tests/1100__data_science_pipelines/1110__test-run-data-science-pipelines-operator-e2e-tests.robot:29]: The tests take a while to run. Could you try to use the `Run And Watch Command` keyword  in Common.robot so we'll be able to see the results in the Console in realtime? (in this or a follow-up PR)

**Review by jgarciao — APPROVED** [review]

**Pull request merged**